auth/README.md

```diff
@@ -0,0 +1,13 @@
+auth plugins
+============
+
+Auth plugins must implement request checking on a HTTP port and path such as `:8080/validate`.
+
+* Valid requests: return 2xx
+* Invalid requests: return non 2xx
+
+It is up to the developer to pick whether a request body is required for validation. For strategies such as [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication), headers are sufficient.
```

Comment:
Does this mean the request body is always sent to the plugin or that there is a configuraiton/flag to toggle that? Perhaps it would be clearer to say 

> The entire original request, headers and body, will be sent to the plugin. It is up to the developer to what data is used for validation. For strategies such as [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication), headers are sufficient.

Reply:
Configuration flag upstream i.e. in the gateway sidecar configuration.